test/all.spec.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var all = require('../').all;
+var tape = require('tape');
+var Handlebars = require('handlebars');
+
+Handlebars.registerHelper(all.name, all);
+
+tape('all', function (test) {
+  var expected = '✔︎';
+  var template;
+  var actual;
+
+  test.plan(3);
+
+  template = Handlebars.compile('{{#all a b c}}✔︎{{/all}}');
+  actual = template({ a: true, b: true, c: true });
+  test.equal(actual, expected, 'Resolves to true when all values are true');
```

Comment:
@tylersticka I'm curious about your thoughts of adding a `truthy` and `falsy` test also? 🙂 

Something like:
```js
  template = Handlebars.compile('{{#all a b c d e f g}}✔︎{{/all}}');
  actual = template({ a: true, b: 14, c: 'false', d: {}, e: [], f: '0', g: -42 });
  test.equal(actual, expected, 'Resolves to true when all values are truthy');
```
```js
  template = Handlebars.compile('{{#all a b c}}{{else}}✔︎{{/all}}');
  actual = template({ a: true, b: undefined, c: true });
  test.equal(actual, expected, 'Resolves to false when any value is falsy');
```

Reply:
@gerardo-rodriguez Good idea!

Reply:
@tylersticka Nice! 😄  

I was referencing this document, in case it's helpful. 😉 

https://developer.mozilla.org/en-US/docs/Glossary/Truthy